apps/judicial-system/api/src/app/modules/case-list/models/caseList.model.ts

```diff
@@ -1,44 +1,35 @@
 import { Field, ID, ObjectType } from '@nestjs/graphql'
 
 import {
-  CaseAppealDecision,
```

Comment:
This model is deprecated like the case list interceptor. This PR removes some fields that are not needed for the current defender tables. The code will be removed when the defender tables have been moved to the new table architecture.